demos/drawer/permanent-drawer-above-toolbar.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDL Drawer Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="../assets/material-design-lite.css.js" charset="utf-8"></script>
+    <style>
+    body, html {
+      display: flex;
+      flex-direction: row;
+      padding: 0;
+      margin: 0;
+      box-sizing: border-box;
+      height: 100%;
+      width: 100%;
+    }
+
+    .toolbar {
+      display: block;
+      width: 100%;
+      height: 56px;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      padding: 0 16px;
+      box-sizing: border-box;
+    }
+
+    @media (min-width: 600px) {
+      .toolbar {
+        height: 64px;
+      }
+    }
+
+    .content {
+      display: inline-flex;
+      flex-direction: column;
+      flex-grow: 1;
+      height: 100%;
+      box-sizing: border-box;
+    }
+
+    main {
+      display: inline-block;
+      padding: 16px;
+      box-sizing: border-box;
+    }
+
+    .mdl-permanent-drawer {
```

Comment:
It is, because it depends on whether you're placing the drawer above or below the toolbar. You'd want a lower z-index than the toolbar for the latter case.